client/js/application.ts

```diff
@@ -2,7 +2,10 @@ enum FormType {
 	Application,
 	Confirmation
 }
-let formType = window.location.pathname.match(/^\/apply/) ? FormType.Application : FormType.Confirmation;
```

Comment:
Whyyyyyyyyy globals. There has to be a better way

Reply:
I literally removed this global